packages/eslint-plugin/src/rules/space-infix-ops.ts

```diff
@@ -0,0 +1,153 @@
+      category: 'Stylistic Issues',
+      recommended: false,
+      extendsBaseRule: true,
+    },
+    fixable: baseRule.meta.fixable,
+    schema: baseRule.meta.schema,
+    messages: baseRule.meta.messages,
+  },
+  defaultOptions: [
+    {
+      int32Hint: false,
+    },
+  ],
+  create(context) {
+    const rules = baseRule.create(context);
+    const sourceCode = context.getSourceCode();
+
+    /**
+     * Find the first non space specified char
+     * @param {TSESTree.token} left The token to the left
+     * @param {TSESTree.Token} right The token to the right
+     * @param {string} op The token to find
+     * @returns {TSESTree.Token|null}
+     * @private
+     */
+    function getFirstNonSpacedToken(
+      left: TSESTree.Token,
+      right: TSESTree.Token,
+      op: string,
+    ): TSESTree.Token | null {
+      const operator = sourceCode.getFirstTokenBetween(
+        left,
+        right,
+        token => token.value === op,
+      );
+      const prev = sourceCode.getTokenBefore(operator!);
+      const next = sourceCode.getTokenAfter(operator!);
+
+      if (
+        !sourceCode.isSpaceBetweenTokens(prev!, operator!) ||
+        !sourceCode.isSpaceBetweenTokens(operator!, next!)
+      ) {
+        return operator;
+      }
+
+      return null;
+    }
+
+    /**
+     * Reports an AST node as a rule violation
+     * @param {TSESTree.Node} mainNode The node to report
+     * @param {TSESTree.Token} culpritToken The token which has a problem
+     * @returns {void}
+     * @private
+     */
+    function report(
+      mainNode: TSESTree.Node,
+      culpritToken: TSESTree.Token,
+    ): void {
+      context.report({
+        node: mainNode,
+        loc: culpritToken.loc,
+        messageId: 'missingSpace',
+        data: {
+          operator: culpritToken.value,
+        },
+        fix(fixer) {
+          const previousToken = sourceCode.getTokenBefore(culpritToken);
+          const afterToken = sourceCode.getTokenAfter(culpritToken);
+          let fixString = '';
+
+          if (culpritToken.range[0] - previousToken!.range[1] === 0) {
+            fixString = ' ';
+          }
+
+          fixString += culpritToken.value;
+
+          if (afterToken!.range[0] - culpritToken.range[1] === 0) {
+            fixString += ' ';
+          }
+
+          return fixer.replaceText(culpritToken, fixString);
+        },
+      });
+    }
+
+    /**
+     * Check if it has an assignment char and report if it's faulty
+     * @param {TSESTree.Node} node The node to report
+     * @returns {void}
+     * @private
+     */
+    function checkForAssignmentSpace(node: TSEnumMember): void {
+      if (!node.initializer) {
+        return;
+      }
+
+      const leftNode = sourceCode.getTokenByRangeStart(
+        sourceCode.getIndexFromLoc(node.id.loc.start),
+      )!;
+      const rightNode = sourceCode.getTokenByRangeStart(
+        sourceCode.getIndexFromLoc(node.initializer.loc.start),
+      )!;
+
```

Comment:
I believe that you can just use `.range[0]` instead of using this method..

Reply:
there is a type mismatch `Argument of type 'number' is not assignable to parameter of type 'LineAndColumnData'`